services/app/src/components/avatar/CustomAvatar.jsx

```diff
@@ -24,6 +24,7 @@ const CustomAvatar = ({ size, imageUrl, playerId, bordered, playerInitials, onEr
     <Avatar 
       size={avatarSizesMap[size]}
       style={getCustomAvatarStyle()}
+      data-testid="profile-picture"
```

Comment:
There will probably be more than 1 custom avatar in the game. You don't want all of them to have the same id